otdfctl/docs/man/migrate/namespaced-policy.md

```
@@ -22,9 +26,41 @@ The parent `migrate` command provides the shared `--commit` and `--interactive`
+## Best practices
+
+1. Before running any migration commands you should take a backup of your database to avoid any potential issues.
+
+2. While we allow you to perform a full migration of policy at once with the multiple scopes, we recommend that you migrate one-by-one in the following order:
+- Actions
+- Subject-Condition-Sets
+- Subject-Mappings
+- Obligation-Triggers
+- Registered-Resources
+
+3. Use `--interactive` mode when migrating policy, this will give you the best chance of success for handling any issues that might occur. In addition, we ask for confirmation for each
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)